x-element.js

```diff
@@ -1385,7 +1387,14 @@ class TemplateEngine {
             string = string.slice(0, -3 - name.length) + `${TemplateEngine.#ATTRIBUTE_PREFIXES.property}${iii}="${name}`;
             state.index = 1; // Accounts for an expected quote character next.
           } else {
-            throw new Error(`Found invalid template string "${string}" at "${string.slice(state.index)}".`);
+            // We try and intuit what authors consider line “1”.
```

Comment:
Seeing the fork here does make me think that we should avoid exceptions like this — if the user chooses to author templates in a certain manner that will have implications for line numbering the library should remain agnostic